jvb/src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1004,24 +1004,36 @@ public IQ healthCheckIqReceived(@NotNull HealthCheckIQ iq)
          * The total number of times an ICE Agent failed to establish
          * connectivity.
          */
-        public AtomicInteger totalIceFailed = new AtomicInteger();
+        public CounterMetric iceFailed = VideobridgeMetricsContainer.getInstance().registerCounter(
```

Comment:
Can you move these from Videobridge.Statistics to IceTransport.Companion?

I see that iceSucceededTcp and iceSucceededRelayed which explains why I've had "fix ice tcp and relayed stats" ticket on my backlog 🤦 . I'll fix that later.

Reply:
The increment should also be internal to IceTransport

Reply:
As an example currently IceTransport calls `connected()`, then `Endpoint`/`Relay` fire `iceSucceeded` and `Conference` increments the `iceSucceeded` metrics. Instead `IceTransport` should increment the metric when it calls `connected()`. 
Basically all the logic for ICE metrics (except for should be in IceTransport